README.md

```diff
@@ -301,3 +301,103 @@ Pretty stnadard, we simply collect all the hashes and the associated types for t
 ### Evaluation
 
 As discusssed in the pinning section, evaluation is done on a fully typechecked and pinned `Expr`ession.
+
+## Developing Inferno scripts with VScode
+
+In a shell go to /inferno and run `nix build .#vscode-inferno-lsp-server`. Then in VScode press `ctrl+shift+P` and run `Install from VSIX`. In the window, navigate to /inferno/result and select `inferno-lsp-server.vsix`.
+
+Do the same after for the VSIX created using `nix build .#vscode-inferno-syntax-highlighting`.
+
+### Regular Inferno
+
+In a shell go to /inferno and run `nix build .#inferno-lsp-server`
+Run `ls -al result`
+Copy the nix/store ... directory to your clipboard. Open VScode, press ctrl + shift + P and search for Open User Settings. Search for Inferno,
+find the inferno LSP extension tab and open it. Paste the directory you copied into the `Path to the inferno-lsp-server executable` field. 
+
+Be sure to append `/bin/inferno-lsp-server` to the end of the directory, then restart VScode.
+
+Create a new file with the .inferno extension. If you begin typing an inferno command such as Array.argmax, the autocomplete box should appear.
+
+Next, if you have the inferno repo cloned into ~/inferno
+
+add
+
+```
+"tasks": [
+        {
+            "label": "inferno",
+            "type": "shell",
+            "command": "cd ~/inferno; nix run .#inferno -- ${file}",
+            "problemMatcher": [],
+            "group": {
+                "kind": "build",
+                "isDefault": true
+            }
+        }
+    ]
+```
+
+to your tasks.json folder in VScode. Otherwise you will need to change `cd ~/inferno;` to the directory where you stored the repo.
+
+You should now be able to build .inferno scripts using `ctrl + shift + B` in VScode
+
+Try saving the following into an .inferno file and compiling it.
+
+```
+let arr = [1, 2, 3, 4] in
+let sum = Array.sum arr in
+sum
+```
+
+The file should compile and output 
+
+`10.0`
+
+### Inferno-ML
+
+in a shell go to /inferno run `nix build .#packages.x86_64-linux.inferno-ml:exe:inferno-ml-lsp-server-ghc924`
```

Comment:
```suggestion
in a shell go to `~/inferno` and run `nix build .#inferno-ml:exe:inferno-ml-lsp-server-ghc925`
```

When you are building a flake output from `packages`, you don't need to specify the `packages.<SYSTEM>` on the command line. Nix does that for you (that's why you can run `nix build .#inferno-lsp-server`, for example, which is also under `packages`)